apps/vapi/GLContextProviderMacOS.mm

```diff
@@ -0,0 +1,46 @@
+#include "GLContextProviderMacOS.h"
```

Comment:
It looks like the source file in Objective-C. It would be good to add a comment to that effect as well as a brief explanation as to why not C++

Reply:
The native macOS libraries are written Objective-C. I've added a comment in that regard.